pages/docs/core-concepts/running-effects.mdx

```diff
@@ -0,0 +1,43 @@
+import { Callout } from "nextra-theme-docs"
+
+# Running Effects
+
+To run an `Effect`, we can use the variety of `run` functions available on the `Effect` module:
+
+```ts twoslash
+import * as Effect from "@effect/io/Effect"
+
+// sync variant
+export const programSync = Effect.runSync(Effect.succeed("Hello, World!"))
+//           ^?
+
+// promise variant
+export const programPromise = Effect.runPromise(
+  //           ^?
```

Comment:
```suggestion
export const programPromise = Effect.runPromise(
//           ^?
```

Reply:
I believe prettier will put these back to where they were automatically. Might have to leave these as is.

Reply:
Yeah this is one of the caveats I mentioned in my other PR. Prettier will format all comments to align with your code (mainly the code underneath the comment). There's no way I'm aware of to turn that off, so you just have to make sure the `?` is aligned correctly after prettier formats and you're good.

Reply:
This isn't an ideal solution, but we could try:
```
{/* prettier-ignore */}
```ts
```

Reply:
I wouldn't want to ignore formatting the entire code block just because of the comment - I don't think the comment formatting is that big of a deal, considering that everything else "just works" with very little config. Not a tradeoff I wish we had to make, but considering everything else works, I think it's fine.

Reply:
Yea - you're right, I just get annoyed by stuff like this 🤣 .